Dockerfile

```diff
@@ -1,11 +1,11 @@
 # We must use a glibc based distro due to embedded python not supporting musl libc for aarch64 (only amd64+musl is supported)
 # see https://github.com/indygreg/python-build-standalone/issues/87
-FROM debian:bullseye-slim
+FROM cgr.dev/chainguard/wolfi-base
 
-RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && rm -rf /var/lib/apt/lists/*
+RUN apk add ca-certificates
```

Comment:
I don't think ca-certificates are required as already in the wolfi-base image follow link https://github.com/chainguard-images/images/blob/4d218924ae71827d1e176229daf98b706874835d/images/wolfi-base/configs/latest.apko.yaml#LL7C9-L7C9

Reply:
See https://github.com/kluctl/kluctl/pull/539